app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,22 @@
+class SessionsController < Devise::SessionsController
+  after_action :log_failed_login, :only => :new
+
+  def create
+    super
+    LoggingHelper.log_event(request,current_user,{event:"user_login"})
```

Comment:
@maxious with the `around_action` thing, just be aware of the controller hierarchy when you  implement it. If you do it in `ApplicationController` _and_ `EditorialController` you may end up doubling up on logging.